content/docs/avalanche-l1s/maintain/add-validator-l1.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Add Validator
+description:  Learn how to add a validator to an Avalanche L1.
+---
+
+
+### Add a Validator to an Avalanche L1
```

Comment:
markdown formatting usually includes blank lines between headings and code blocks